src/components/views/dashboard/OverviewChart.js

```diff
@@ -1,27 +1,49 @@
+import classNames from 'classnames';
 import { useEffect, useState } from 'react';
-import { Line } from 'react-chartjs-2';
+import { Bar, Line } from 'react-chartjs-2';
+import {
+  MdShowChart,
+  MdStackedBarChart,
+  MdStackedLineChart,
+} from 'react-icons/md';
 
 import {
   DAY,
   parseAndCondenseStackedTimeSeriesData,
+  parseSearchReturn,
 } from '../../../services/search';
 import {
   formatXAxisLabels,
   splitDayAndNightDataSets,
 } from '../../../utils/Utils';
 import { tooltipPlugin } from '../../common/graphPlugins';
 
-export default function OverviewChart({ timeIncrement, siteData }) {
+export default function OverviewChart({
+  timeIncrement,
+  startDate,
+  endDate,
+  overviewData,
+  sitesData,
+}) {
   const [loading, setLoading] = useState(true);
   const [dayData, setDayData] = useState([]);
   const [nightData, setNightData] = useState([]);
+  const [graphType, setGraphType] = useState('overview');
+
+  const dateFormat = {
```

Comment:
We should match this to what's presented in alerts, reports (Jan 12, 24 01:16 PM vs 1/12/24, 01:16PM)